knexfile.js

```diff
@@ -0,0 +1,46 @@
+// Update with your config settings.
+
+module.exports = {
+
+  development: {
+    client: 'pg',
+    connection: 'postgres://localhost/palette_picker',
+    migrations: {
+      directory: './db/migrations'
+    },
+    useNullAsDefault: true
+  },
+
+  staging: {
+    client: 'postgresql',
+    connection: {
+      database: 'my_db',
+      user:     'username',
+      password: 'password'
+    },
+    pool: {
+      min: 2,
+      max: 10
+    },
+    migrations: {
+      tableName: 'knex_migrations'
+    }
+  },
+
```

Comment:
When we get into testing we will want to reconfigure the knex file, but for now this looks good!